src/EventExport/Format/HTML/Uitpas/EventInfo/CultureFeedEventInfoService.php

```diff
@@ -131,7 +131,7 @@ private function getUitpasAdvantagesFromEvent(CultureFeed_Uitpas_Event_CultureEv
         $advantages = [];
 
         if ($this->pointCollecting->isSatisfiedBy($event)) {
-            $advantages[] = EventAdvantage::POINT_COLLECTING;
+            $advantages[] = EventAdvantage::POINT_COLLECTING();
```

Comment:
This used to store a `string` value, now it will store a `EventAdvantage` object.

Reply:
I noticed that the type hinting was wrong so I fixed that.